.github/workflows/ci.yml

```diff
@@ -207,6 +207,9 @@ jobs:
       run:
         shell: msys2 {0}
     steps:
+      - name: Prepare git config
+        run: git config --system core.autocrlf false
```

Comment:
Maybe we want a comment why this is done?

Is it worth having some kind of regression test to make sure --version seem ok?